kroxylicious/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -84,10 +85,12 @@ public class KafkaProxyFrontendHandler
     private String sniHostname;
 
     private ChannelHandlerContext inboundCtx;
-    // The message buffered while we connect to the outbound cluster
-    // There can only be one such because auto read is disabled until outbound
+
+    // Messages buffered while we connect to the outbound cluster
+    // The size should be limited because auto read is disabled until outbound
```

Comment:
Note I haven't put any depth limit on this because we disable inbound autoread, it should only be filled with as many messages as are pulled in from the socket by the initial read.

Reply:
We could opt to put some limit on this though.